.github/workflows/AirspeedVelocity.yml

```
@@ -0,0 +1,78 @@
+name: Benchmark a Pull Request
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  pull-requests: write
+
+jobs:
+    generate_plots:
```

```suggestion
    benchmark:
```
IMO the name is not quite appropriate because it does more than generating plots.